### PR TITLE
Add panic logs check

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 

Use a goroutine to check panic logs from loki.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: cmd/util/suit.go [hunk 1]
```diff
@@ -90,6 +92,8 @@ func (suit *Suit) Run(ctx context.Context) {
 		suit.NemesisGens,
 		suit.ClientRequestGen,
 		suit.VerifySuit,
+		loki.NewLokiClient(fixture.Context.LokiAddress,
```
Comment (mahjonp) [inline cmd/util/suit.go]: How about use a interface here?

File: pkg/control/control.go [hunk 2]
```diff
@@ -54,6 +57,7 @@ func NewController(
 	nemesisGenerators []core.NemesisGenerator,
 	clientRequestGenerator func(ctx context.Context, client core.Client, node clusterTypes.ClientNode, proc *int64, requestCount *int64, recorder *history.Recorder),
 	verifySuit verify.Suit,
+	lokiCli *loki.LokiClient,
```
Comment (mahjonp) [inline pkg/control/control.go]: How about use an interface here?

File: pkg/test-infra/fixture/fixture.go [hunk 3]
```diff
@@ -16,9 +16,8 @@ package fixture
 import (
 	"flag"
 	"net/http"
-	"time"
-
 	_ "net/http/pprof" // pprof
+	"time"
```
Comment (mapleFU) [inline pkg/test-infra/fixture/fixture.go]: Just put them together is OK

File: pkg/control/control.go (hunk 2 above)
Comment (yeya24) [reply pkg/control/control.go]: I think it is ok to use lokiClient for now. But let me add a todo. What do you think

**Pull request merged**